src/index.js

```diff
@@ -31,6 +36,9 @@ Dotdotdot.prototype.dotdotdot = function(container) {
     });
   }
 };
+Dotdotdot.prototype.update = function() {
```

Comment:
I've checked docs for `forceUpdate` and it is safer to leave it as it is.
Force update is expecting a callback as its first param, and doing it as you proposed will pass there an event. It might lead to unexpected behavior and should be avoided I guess.